kraken/plugins/test_pod_plugin.py

```diff
@@ -0,0 +1,175 @@
+import random
+import re
+import string
+import threading
+import unittest
+
+from arcaflow_plugin_sdk import plugin
+from kubernetes.client import V1Pod, V1ObjectMeta, V1PodSpec, V1Container, ApiException
+
+from kraken.plugins import pod_plugin
+from kraken.plugins.pod_plugin import setup_kubernetes, KillPodConfig, PodKillSuccessOutput
+from kubernetes import client
+
+
+class KillPodTest(unittest.TestCase):
```

Comment:
are we able to move this test file to a test folder at the top level to not have this plugin folder become too big once we keep adding more and more plugins

Reply:
sounds good. I'll create an issue on jira (or github) that it would be nice to move that out but not sure how currently